front/src/components/header/Header.tsx

```diff
@@ -187,6 +188,7 @@ const mapStateToProps: MapStateToProps<StateProps, {}, AppState> = (
   state: AppState
 ): StateProps => ({
   isLoggedIn: selectIsLoggedIn(state),
+  email: state.user.email,
```

Comment:
Do you mind creating a selector, similar to `selectIsLoggedIn` here?

Reply:
no problem, so what kind of selector do you want?

Reply:
It can look just like `selectIsLoggedIn` in `user/selectors.ts`

Reply:
ok, is there a preference for the name?

Reply:
Maybe `selectUserEmail`